objection_engine/v4/font_constants.py

```diff
@@ -0,0 +1,36 @@
+from enum import IntEnum
```

Comment:
Why couldn't we use the old file instead of copy/pasting it?

Reply:
Looking at the two files, the only difference I see is that the "Igiari-cyrillic.ttf" font file was added as highest priority, and is stored in the `assets_v4` folder instead of the old `assets` folder. It was a while since I worked on that part of the code, but my guess is that I wanted to avoid making any modifications to the old engine so it'd continue to work even if my version had bugs.

If you'd prefer, we could get rid of this mostly-copied file and add the "Igiari-cyrillic.ttf" font to the existing "font_constants.py" file, so that both engines benefit from the font change.

Reply:
LEt's keep them isolated from now as my long term plan is to just replace v3 completely, so this way is easy to just move all files form v4 alltogether